src/arrayMethodLastIndexOf.js

```diff
@@ -5,7 +5,23 @@
  */
 function applyCustomLastIndexOf() {
   [].__proto__.lastIndexOf2 = function(searchElement, fromIndex) {
-    // write code here
+    const startIndex = fromIndex || this.length - 1;
```

Comment:
краще добавляти default значення для параметра
```suggestion
    const startIndex = fromIndex;
```